recipes/weldx/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "weldx" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #git_url: /home/marscher/sources/weldx
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 707fa793d92089e976865870dc603ce9cf34cea4c9af66fd6aff9a6c7c3c79d1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+
+requirements:
+  host:
+    - pip
+    - setuptools_scm
+    - python
+  run:
+    - asdf >=2.7
+    - boltons
+    - bottleneck
+    - fs
+    - jinja2
+    - ipywidgets
+    - matplotlib-base
+    - networkx
+    - numpy >=1.18
+    - pandas >=1.0
+    - pint >=0.11
+    - python >=3.6
+    - scipy <1.6
+    - sympy >=1.6
+    - xarray >=0.15
+
+test:
+  imports:
+    - weldx
+
+about:
+  home: https://www.bam.de/Content/EN/Projects/WelDX/weldx.html
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Python API for the WelDX file format and standard"
+  description: |
+    Scientific welding data covers a wide range of physical domains and timescales and are measured using various different sensors. Complex and highly specialized experimental setups at different welding institutes complicate the exchange of welding research data further.
+
+    The WelDX research project aims to foster the exchange of scientific data inside the welding community by developing and establishing a new open source file format suitable for documentation of experimental welding data and upholding associated quality standards. In addition to fostering scientific collaboration inside the national and international welding community an associated advisory committee will be established to oversee the future development of the file format. The proposed file format will be developed with regard to current needs of the community regarding interoperability, data quality and performance and will be published under an appropriate open source license. By using the file format objectivity, comparability and reproducibility across different experimental setups can be improved.
+  doc_url: https://weldx.readthedocs.io
+  dev_url: https://github.com/BAMWelDX/weldx/
+
+extra:
+  recipe-maintainers:
+    - marscher
+    - vhirtham
```

Comment:
@vhirtham, are you happy to be listed as a recipe maintainer here?

Reply:
@chrisburr yes, that's okay